scripts/generate-app-config/generateEvidenceConfig.js

```diff
@@ -17,9 +17,9 @@ const getScreenshotPatterns = generatePagesOptions => (
       entryPoint: `${rootPath}/packages/[^/]*/(test|tests)/wdio/__snapshots__/reference/`,
     });
     acc.push({
-      pattern: `${rootPath}/{test,tests}/wdio/__snapshots__/reference/**/*.png`,
+      pattern: `${rootPath}/{test,tests}/wdio/**/__snapshots__/reference/**/*.png`,
       // build out a regex for the entrypoint mask.
-      entryPoint: `${rootPath}/(test|tests)/wdio/__snapshots__/reference/`,
+      entryPoint: `${rootPath}/(test|tests)/wdio/.*__snapshots__/reference/`,
```

Comment:
`/.*__snapshots__/` -> `/**/__snapshots__/`

Reply:
.* means matches any character zero or unlimited times. Not sure if /**/ is valid. It is valid for file paths but not for regex.

Reply:
hmmm... yeah but I'm not sure that will look beyond the immediate directory layer. Looking at this though, I would expect the same regex change for both patterns.

Reply:
The pattern is a glob pattern and the entryPoint is regex that's why they are different. 
/**/ is not a valid expression in regex and .* is not a valid expression in glob.
When i tested locally, it was able to give snapshots for 
```
wdio/__snapshots__/
wdio/subfolder/__snapshots__/
```
We can also test that entryPoint on [regex101](https://regex101.com/) to confirm.

Reply:
@parvezalisha  Ahhh I see 😄 Looks good to me!

Saved Regex example: https://regex101.com/r/0PvCoy/1